index.html

```diff
@@ -23,14 +22,29 @@
       <h1 id="header-text">Web Developer</h1>
     </div>
 
-    <div id="About">
-        <p>This is where my "About" section will go.</p>
-        <p>I still need to figure out how all of it is going to look.</p>
-    </div>
+    <section id="about-section">
+      <img id="profile-image" src="https://i.imgur.com/w5QpEe3.jpg" alt="An image of my face.">
+      <div id="blank-column"></div>
+      <div id="about-text">
+        <h2>About Me</h2>
+        <p>I'm an aspiring front end developer currently putting together my portfolio site while documenting my coding journey <a href="https://rodericksang.com/blog">in my blog</a> which was built using Jekyll.</p>
+        <p>My interests include video games, playing music (guitar and saxophone), and also game design/game development.</p>
+        <p>I'm a lover of pixel art, as well. The gif used in the header is created by my favorite artist Waneella. You can check out their art <a href="https://waneella.tumblr.com/">on their Tumblr.</a> I cannot recommend it enough.</p>
+        <p>If you are a more experienced developer and have any pointers or pieces of advice you're wanting to share, please do so! My email address is roderick@protonmail.com.</p>
+      </div>
+    </section>
 
-    <div id="Work">
+    <section id="work">
       Something will go here someday soon.
-    </div>
+    </section>
+
+    <footer class="footer">
+      <p>Roderick Sang &copy; 2019</p>
+      <ul class="footer-links">
```

Comment:
I notice you're using a class here and ids other places - is there a reason for that? Just curious, it's not a bad thing necessarily.

Also just FYI, it's pretty common in front end development these days to use only classes and not ids (if possible), mostly because they are re-usable and easier to combine.